<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterMaterialsGUI</class>
 <widget class="QWidget" name="FilterMaterialsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilterMaterialsGUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="filteredMaterials_listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="size_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Microfragments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="microfragments_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="filter_pushButton">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
